submissions/webdevtoday/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,325 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Task</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main class="Main">
```

Comment:
First time in my life I see class name started with a capital letter. Good practice use lowercase letters for class names.)

Reply:
Hi! Thank you! I'll fix it